InvenTree/label/templates/label/multipage_label_base.html

```diff
@@ -0,0 +1,49 @@
+{% load l10n %}
+
+<head>
+    <style>
+        {% localize off %}
+        @page {
+            size: {{ page_width }}mm {{ page_height }}mm;
+            margin-top: 0mm;
+            margin-bottom: 0mm;
+            margin-left: 0mm;
+            margin-right: 0mm;
```

Comment:
Is there a reason why you provide them individually?

Reply:
No reason, I am just dumb, I am going to fix this.